### PR TITLE
Do not check status of processes on non existing container

### PR DESCRIPTION
**Pull request by kokosing**
Do not check status of processes on non existing container

### CODE REVIEW

File: presto-product-tests/bin/run_on_docker.sh [hunk 1]
```diff
@@ -22,12 +22,8 @@ function retry() {
   return ${EXIT_CODE}
 }
 
-function hadoop_master_container(){
-  environment_compose ps -q hadoop-master
```
Comment (findepi) [inline presto-product-tests/bin/run_on_docker.sh]: See https://github.com/prestosql/presto/blob/bec0121e1ea4620ecb65effdbce96d688ad051be/presto-hive-hadoop2/bin/common.sh#L21

**Comment by kokosing** [comment]
@findepi Updated. Please take a look again.

File: presto-hive-hadoop2/bin/common.sh [hunk 2]
```diff
@@ -28,7 +28,7 @@ function hadoop_master_ip() {
 }
 
 function check_hadoop() {
-  HADOOP_MASTER_CONTAINER=$(hadoop_master_container)
+  HADOOP_MASTER_CONTAINER=$(hadoop_master_container | grep .)
```
Comment (findepi) [inline presto-hive-hadoop2/bin/common.sh]: i'd add `| grep .` inside `hadoop_master_container` function. Any reason not to?

File: presto-product-tests/bin/run_on_docker.sh (hunk 1 above)
Comment (findepi) [reply presto-product-tests/bin/run_on_docker.sh]: Please revert. I use this function :)

**Pull request merged**